Dockerfile

```diff
@@ -150,22 +150,6 @@ ENV HOME="/home/smriprep"
 # Installing dev requirements (packages that are not in pypi)
 WORKDIR /src/
 
-# Precaching atlases
-COPY setup.cfg setup.cfg
-RUN pip install --no-cache-dir "$( grep templateflow setup.cfg | xargs )" && \
-    python -c "from templateflow import api as tfapi; \
-               tfapi.get(['MNI152Lin', 'MNI152NLin2009cAsym', 'OASIS30ANTs'], suffix='T1w'); \
-               tfapi.get(['MNI152Lin', 'MNI152NLin2009cAsym', 'OASIS30ANTs'], desc='brain', suffix='mask'); \
-               tfapi.get('OASIS30ANTs', resolution=1, desc='4', suffix='dseg'); \
-               tfapi.get(['OASIS30ANTs', 'NKI'], resolution=1, label='brain', suffix='probseg'); \
-               tfapi.get(['MNI152NLin2009cAsym', 'OASIS30ANTs', 'NKI'], resolution=1, \
-                         desc='BrainCerebellumExtraction', suffix='mask'); \
-               tfapi.get(['MNI152NLin2009cAsym', 'OASIS30ANTs', 'NKI'], resolution=1, \
-                         desc='BrainCerebellumRegistration', suffix='mask');" && \
-    rm setup.cfg && \
-    find $HOME/.cache/templateflow -type d -exec chmod go=u {} + && \
-    find $HOME/.cache/templateflow -type f -exec chmod go=u {} +
```

Comment:
I'm not sure why we wouldn't want to bundle these in images.

Reply:
I think mounting the templateflow folder from outside is more effective for sMRIPrep. I probably would not propagate this to fMRIPrep.

Pros:
- The image is not rebuilt every time there's a change on the setup.cfg file (any change, not just templateflow's pin).
- The image is a bit lighter and the dockerfile simpler
- It is easier for nipype to use cached results, and we might want to roll it back to timestamp hashing at some point.
- The pybids pin was ineffective because of the installation of templateflow, which already installed 0.9.4 as a dependency. When smriprep was installed, it identified pybids was available regardless of the pin.
- Saves build time in the long run, as templateflow will identify all these files as cached.

Cons:
- Time-cost of this PR, perhaps?